gradle/release.gradle

```diff
@@ -30,6 +30,24 @@ task sourceJar(type: Jar) {
     classifier "sources"
 }
 
+allprojects { proj ->
+    afterEvaluate {
+        validatePublishedAbis(proj)
+    }
+}
+
+// Throws an exception if the build is not targeting the required ABIs for publishing
+private void validatePublishedAbis(proj) {
+    if (proj.android.defaultConfig.ndk.abiFilters != null) {
+        for (String abi : ["arm64-v8a", "armeabi", "armeabi-v7a", "x86", "x86_64"]) {
```

Comment:
I've just realised that checking the ABIs like this will break the Unity build: https://github.com/bugsnag/bugsnag-unity/blob/master/Rakefile#L83

Perhaps there's something in the Gradle DSL that allows us to check the version of the NDK used instead, and verify that it's r16b? https://google.github.io/android-gradle-dsl/current/com.android.build.gradle.LibraryExtension.html